services/src/util/config.rs

```diff
@@ -101,7 +103,8 @@ pub trait ConfigElement {
 #[derive(Debug, Deserialize)]
 pub struct Web {
     pub bind_address: String,
```

Comment:
why is this not a url?

Reply:
bind address is a socket address as discussed